### ISSUE TITLE
Odoo Project No one can see the project or tasks

### ISSUE THREAD

**Issue by aario** [odoo/odoo#16747]
Impacted versions:
10 Community edition
Steps to reproduce:
As admin create a project a task a user an employee and set the project to be visible by all employees. As the user login and you cannot see any project or tasks. Even setting the new user as manager of the project doesn't make the project visible to him.
Current behavior:
User cannot see any project or tasks created by administrator.
Expected behavior:
User should see tasks assigned to him.
Video/Screenshot link (optional):

**Comment by Yenthe666** [comment]
Hi @aario,

Thanks for your report!
I've just tested this on the latest Odoo 10 and this works without any problem.
I made a new project named "YenProject" (and set it available to all employees), made a new task named "YenTask" and a user "Yenthe" with the administrator user. After logging in with the new user "Yenthe" I can see the project and the task:
![image](https://cloud.githubusercontent.com/assets/6352350/25651768/10917586-2fe5-11e7-9369-755df6b989a5.png)

Could you try to reproduce this on a runbot (runbot.odoo.com)?
Please try updating your own Odoo too.

Regards,
Yenthe

**Comment by aario** [comment]
I found the reason for the problem. It was because our Odoo has multiple company and the project was created in one company while the user was on another company (The company drop down on top right corner). Once the user selects the relevant company tasks and projects appear.
Now my problem is how to move a project from company to company. I found another topic/post/issue report somewhere on the internet about this but no answer was there. Is it possible to move a project from company to company through the website or once should do it the hard way (Through database)?
